waspc/data/Generator/templates/server/src/middleware/globalMiddleware.ts

```diff
@@ -0,0 +1,46 @@
+{{={= =}=}}
+import express from 'express'
+import cookieParser from 'cookie-parser'
+import logger from 'morgan'
+import cors from 'cors'
+import helmet from 'helmet'
+
+import config from '../config.js'
+
+{=# globalMiddlewareConfigFnDefined =}
+{=& globalMiddlewareConfigFnImportStatement =}
+{=/ globalMiddlewareConfigFnDefined =}
+{=^ globalMiddlewareConfigFnDefined =}
+const {=& globalMiddlewareConfigFnImportAlias =} = (m: MiddlewareConfig) => m
+{=/ globalMiddlewareConfigFnDefined =}
+
+export type MiddlewareConfig = Map<string, express.RequestHandler>
```

Comment:
Kept as a `Map` instead of `ReadonlyMap` just to make it easier to work with in those `MiddlewareConfigFn` for users. We clone the defaults so they are not impacted.